### PR TITLE
Fix template apps

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?179392
@benrudolph 

cc @TylerSheffels

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/migrate_template_apps_form_ids.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+            app_doc = update_unique_ids(app_doc)
+
+        return Application.wrap(app_doc) if should_save else None
+
+    def get_app_ids(self):
+        q = AppES().created_from_template(True).is_build(False).fields('_id')
+        results = q.run()
+        return [app['_id'] for app in results.hits]
+
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/migrate_template_apps_form_ids.py]: W391 blank line at end of file

File: corehq/apps/app_manager/util.py [hunk 2]
```diff
@@ -515,3 +516,32 @@ def get_cloudcare_session_data(domain_name, form, couch_user):
...
+    from corehq.apps.app_manager.models import FormBase, form_id_references, jsonpath_update
+
+    app_source = deepcopy(app_source)
+
+    def change_unique_id(form):
+        unique_id = form['unique_id']
+        new_unique_id = FormBase.generate_id()
+        form['unique_id'] = new_unique_id
+        if ("%s.xml" % unique_id) in app_source['_attachments']:
+            app_source['_attachments']["%s.xml" % new_unique_id] = app_source['_attachments'].pop("%s.xml" % unique_id)
```
Comment (dimagimon) [inline corehq/apps/app_manager/util.py]: E501 line too long (119 > 115 characters)

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by benrudolph** [comment]
cool looks good

**Pull request merged**